src/main/java/uk/gov/hmcts/reform/civil/helpers/bundle/BundleRequestMapper.java

```diff
@@ -868,6 +871,116 @@ private BundlingRequestDocument buildBundlingRequestDoc(String docName, Document
+
+        filterGenericDisclosure(caseData.getDocumentSkeletonArgument());
+        filterGenericDisclosure(caseData.getDocumentSkeletonArgumentApp2());
+        filterGenericDisclosure(caseData.getDocumentSkeletonArgumentRes());
+        filterGenericDisclosure(caseData.getDocumentSkeletonArgumentRes2());
+
+        //CaseData
+        filterGenericCaseData(caseData.getDefendantResponseDocuments());
+        filterGenericCaseData(caseData.getClaimantResponseDocuments());
+        filterGenericCaseData(caseData.getSystemGeneratedCaseDocuments());
+
+        //Witness
+        filterGenericWitness(caseData.getDocumentWitnessStatement());
+        filterGenericWitness(caseData.getDocumentWitnessStatementApp2());
+        filterGenericWitness(caseData.getDocumentWitnessStatementRes());
+        filterGenericWitness(caseData.getDocumentWitnessStatementRes2());
+        filterGenericWitness(caseData.getDocumentWitnessSummary());
+        filterGenericWitness(caseData.getDocumentWitnessSummaryApp2());
+        filterGenericWitness(caseData.getDocumentWitnessSummaryRes());
+        filterGenericWitness(caseData.getDocumentWitnessSummaryRes2());
+        filterGenericWitness(caseData.getDocumentHearsayNotice());
+        filterGenericWitness(caseData.getDocumentHearsayNoticeApp2());
+        filterGenericWitness(caseData.getDocumentHearsayNoticeRes());
+        filterGenericWitness(caseData.getDocumentHearsayNoticeRes2());
+
+        //Expert
+        filterGenericExpert(caseData.getDocumentExpertReport());
+        filterGenericExpert(caseData.getDocumentExpertReportApp2());
+        filterGenericExpert(caseData.getDocumentExpertReportRes());
+        filterGenericExpert(caseData.getDocumentExpertReportRes2());
+        filterGenericExpert(caseData.getDocumentQuestions());
+        filterGenericExpert(caseData.getDocumentQuestionsApp2());
+        filterGenericExpert(caseData.getDocumentQuestionsRes());
+        filterGenericExpert(caseData.getDocumentQuestionsRes2());
+        filterGenericExpert(caseData.getDocumentAnswers());
+        filterGenericExpert(caseData.getDocumentAnswersApp2());
+        filterGenericExpert(caseData.getDocumentAnswersRes());
+        filterGenericExpert(caseData.getDocumentAnswersRes2());
+        filterGenericExpert(caseData.getDocumentJointStatement());
+        filterGenericExpert(caseData.getDocumentJointStatementApp2());
+        filterGenericExpert(caseData.getDocumentJointStatementRes());
+        filterGenericExpert(caseData.getDocumentJointStatementRes2());
+
+    }
+
+    private void filterGenericExpert(List<Element<UploadEvidenceExpert>> uploadEvidenceExpert) {
+
+        if (nonNull(uploadEvidenceExpert)) {
+            uploadEvidenceExpert.stream().filter(caseDocumentElement -> (nonNull(caseDocumentElement.getValue().getExpertDocument().getCategoryID())
+                && !caseDocumentElement.getValue().getExpertDocument().getCategoryID().equals("UnbundledFolder")));
```

Comment:
I think UnbundledFolder could be a constant at the top of the class that is then used in every method.

Reply:
yes!!! done.